src/api/libcellml/reset.h

```diff
@@ -57,107 +57,112 @@ class LIBCELLML_EXPORT Reset: public OrderedEntity
     VariablePtr variable() const;
 
     /**
-     * @brief Add a child when to this @c Reset.
+     * @brief Set the @c TestVariable for this @c Reset.
      *
-     * Add a copy of the given @p when as a child @c When of this @c Reset.
+     * Sets the @c TestVariable for this @c Reset.
```

Comment:
Use the infinitive, i.e. "Set" rather than "Sets". (Check the rest of the PR for similar occurrences.)

Reply:
Done :)